tests/acceptance/brocfile-smoke-test-slow.js

```diff
@@ -183,9 +183,8 @@ describe('Acceptance: brocfile-smoke-test', function () {
 
     let ember = path.join('.', 'node_modules', 'ember-cli', 'bin', 'ember');
 
-    let error = await expect(runCommand(ember, 'test', '--filter=jshint')).to.eventually.be.rejected;
-
-    expect(error.output.join('')).to.include('Error: No tests matched the filter "jshint"');
+    let response = await expect(runCommand(ember, 'test', '--filter=jshint')).to.eventually.be.ok;
+    expect(response.output.join('')).to.include('tests 0');
```

Comment:
Hi :)

I'm trying to connect the dots between this and https://github.com/qunitjs/qunit/issues/1651.

Is the response output here by any chance based on the DOM's `Node.textContent` in some way? It appears not, but if not then I'm not exactly sure why this changed.

Reply:
`ember test` runs the qunit browser runner through `testem` and would reject when running and having a failing test.